### ISSUE TITLE
Ctrl+R should reload the folder (not the app)

### ISSUE THREAD

**Issue by koppor** [aleksey-hoffman/sigma-file-manager#306]
When I press <kbd>Ctrl</kbd>+<kbd>R</kbd>, my expectation is that the folder is reloaded. However, the app is reloaded and I am back in the home screen.

**Comment by aleksey-hoffman** [comment]
Good point, mate. I'm planning to make all shortcuts configurable in v2
